src/TurbulenceConvection/EDMF_functions.jl

```diff
@@ -368,7 +368,7 @@ function set_edmf_surface_bc(
             prog_up[i].ρaq_liq[kc_surf] = prog_up[i].ρarea[kc_surf] * q_liq_surf
             prog_up[i].ρaq_ice[kc_surf] = prog_up[i].ρarea[kc_surf] * q_ice_surf
         end
-        prog_up_f[i].ρaw[kf_surf] = ρ_f[kf_surf] * w_surface_bc(surf)
```

Comment:
Were we missing area fraction here before?

Reply:
> Were we missing area fraction here before?

yes but `w_surface_bc(surf) = 0` we are lucky ;-)

Reply:
phew...